src/Nethermind/Nethermind.Synchronization/Peers/SyncPeerPool.cs

```diff
@@ -225,8 +238,10 @@ internal IEnumerable<SyncPeerAllocation> ReplaceableAllocations
         }
 
         public int PeerCount => _peers.Count;
+        public int PriorityPeerCount => _peers.Values.Count(p => p.SyncPeer.IsPriority.Equals(true));
```

Comment:
If used often maybe should be separate counter?

Reply:
Except of logger in debug mode, it's used only in one place, only if there are priority peers enabled. Adding separate counter will need adding plenty of increments/decrements with checks (if peer is priority then increment/decrement). Not worth imo

Reply:
I think peers are added only in AddPeer and removed only in RemovePeer, that's not that many checks. and with Interlocked you would have threadsafety